src/lib/features/project/fake-project-owners-read-model.ts

```diff
@@ -0,0 +1,35 @@
+import type { IProjectWithCount } from '../../types';
+import type {
+    IProjectOwnersReadModel,
+    IProjectWithCountAndOwners,
+    ProjectOwnersDictionary,
+} from './project-owners-read-model.type';
+
+const T = {
+    ROLE_USER: 'role_user',
+    GROUP_ROLE: 'group_role',
+    ROLES: 'roles',
+    USERS: 'users',
+};
+
+export class FakeProjectOwnersReadModel implements IProjectOwnersReadModel {
+    static addOwnerData(
```

Comment:
I feel like we can remove this method from the fake model as well? If we're only ever doing the same thing, might as well just put that directly into addOwners?